docs/appkit/features/smart-accounts.mdx

```diff
@@ -13,40 +13,74 @@ This update ensures a seamless and efficient experience for managing digital ass
 
 <Button name="Try Demo" url="https://appkit-lab.reown.com/" />
 
-## Get Started
-
-<Wrapper
-  type="large"
-  fit={false}
-  items={[
-    {
-      name: 'React',
-      type: 'react',
-      description: 'Get started with AppKit in React.',
-      icon: reactLogo,
-      href: '../react/core/smart-accounts'
-    },
-    {
-      name: 'Next.js',
-      type: 'next',
-      description: 'Get started with AppKit in Next.js.',
-      icon: nextjsLogo,
-      href: '../next/core/smart-accounts',
-      isWhite: true
-    },
-    {
-      name: 'Vue',
-      type: 'vue',
-      description: 'Get started with AppKit in Vue.',
-      icon: vueLogo,
-      href: '../vue/core/smart-accounts'
-    },
-    {
-      name: 'JavaScript',
-      type: 'javascript',
-      description: 'Get started with AppKit in JavaScript.',
-      icon: javascriptLogo,
-      href: '../javascript/core/smart-accounts'
-    }
-  ]}
-/>
+### Deployment
+
+Smart Accounts are deployed alongside the first transaction. Until deployment, a precalculated address, known as the counterfactual address, is displayed. Despite not being deployed, the account can still sign using 6492 signatures.
+
+### Supported Networks
+
+Smart Accounts are available on the following networks:
+
+- Base Sepolia
+- BSC (Binance Smart Chain)
+- Fraximal
+- Linea
+- Mode
+- Optimism
+- Polygon
+- Polygon Mumbai
+- Sepolia
```

Comment:
Can we update this? It supports all EVM chains that are listed on Viem's package. You can refer to the doc page under "Features" -> "Email and Social Login" for the text. You can add the same text here as well

Reply:
For smart accounts we need to use Pimlico's supported chains, not viem as we depend on premissionless for them!